### DISCUSSION TITLE
Quick Settings should appear during pause

### DISCUSSION THREAD

**Discussion by Bubbleman532** [ppy/osu#25541] [Ideas] [answered]
In stable, the visual settings section at the bottom of the screen appears during a pause time. It'd be nice if the same happened with the visual settings and input settings sections in lazer for quick adjustment of background dim or beatmap hitsounds without having to retry the map.

If the intended solution to this is to use Ctrl+O, then disregard this, however I seem to remember that the visual settings section in stable doesn't change the global options when used and it would be nice to have a similar functionality (settings don't have to _save_ per-map, I'm aware how large the scope of that would be).

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/3853
